src/index.js

```diff
@@ -0,0 +1,199 @@
+};
+
+const isCelcius = () => {
+  if (state.tempUnitToggle.checked) {
+    state.tempValue.textContent = Math.floor((state.temperature - 32) * 5 / 9);
+    return true;
+  }
+  else {
+    tempValue.textContent = state.temperature;
+    return false;
+  }
+};
+
+const changeColorBasedOnTemp = () => {
+  state.tempValue.classList.remove('red', 'orange', 'yellow', 'green', 'teal');
+
+  if (state.temperature >= 80) {
+    state.tempValue.classList.add('red');
+  } else if (state.temperature >= 70 && state.temperature <= 79) {
+    state.tempValue.classList.add('orange');
+  } else if (state.temperature >= 60 && state.temperature <= 69) {
+    state.tempValue.classList.add('yellow');
+  } else if (state.temperature >= 50 && state.temperature <= 59) {
+    state.tempValue.classList.add('green');
+  } else if (state.temperature <= 49) {
+    state.tempValue.classList.add('teal');
+  }
+  //TOGGLE LANDSCAPE
+  state.tree[0].classList.toggle('show', state.temperature <= 49);
+  state.garden.classList.toggle('snowy-sky', state.temperature <= 49);
+  state.ground[0].classList.toggle('snowy', state.temperature <= 49);
+
+  state.tree[2].classList.toggle('show', state.temperature >= 50 && state.temperature <= 64);
+  state.ground[0].classList.toggle('cloudy', state.temperature >= 50 && state.temperature <= 64);
+  state.garden.classList.toggle('cloudy-sky', state.temperature >= 50 && state.temperature <= 64);
+
+  state.tree[1].classList.toggle('show', state.temperature >= 65 && state.temperature <= 75);
+
+  state.tree[3].classList.toggle('show', state.temperature > 75);
+  state.ground[0].classList.toggle('sunny', state.temperature > 75);
+};
+
+const changeSkyscape = () => {
+  state.sky[0].textContent = '';
+  // CLOUDY
+  if (state.skySelect.value === 'cloudy') {
+    state.sky[0].textContent = '🌾🍃🌾';
+  }
+  // SUNNY
+  if (state.skySelect.value === 'sunny') {
+    state.sky[0].textContent = '☀️😎☀️';
+  }
+
+  // SNOWY
+  if (state.skySelect.value === 'snowy') {
+    state.sky[0].textContent = '❄️🥶❄️';
+  }
+
+  // RAINY
+  if (state.skySelect.value === 'rainy') {
+    state.sky[0].textContent = '💧☔️💧';
+  }
+
+  //CLOUDY or RAINY
+  state.cloud[0].classList.toggle('show', state.skySelect.value === 'cloudy' || state.skySelect.value === 'rainy');
+  state.cloud[1].classList.toggle('show', state.skySelect.value === 'cloudy' || state.skySelect.value === 'rainy');
+};
+
+const changeCityName = () => {
+  state.headerCityName.textContent = state.input.value;
+};
+
+const resetCity = () => {
+  state.input.value = '';
+  state.headerCityName.textContent = 'Primm';
+  updateWithRealtimeWeather()
+};
+
+const parseRegion = (region) => {
+  region = region.split(',');
+  region = `${region[0]}, ${region[region.length - 2]}, ${region[region.length - 1]}`;
+  return region;
+};
+
+const getLatLon = async () => {
+  const response = await axios.get('https://weather-report-proxy-msis.onrender.com/location', { params: { q: state.headerCityName.textContent } });
```

Comment:
Prefer to store the server portion of the URL in a variable so that in order to point the UI at either a development or deployed backend, we only need to change a single value rather than each API URL.